hw9

irbis45 opened the pull request:
# локализация
# получение локали по умолчанию из браузера
# попытка реализовать middleware на PSR-7, PSR-15. На Laravel 12 нормально не работает

ruslan1984 commented:
В старом варианте один класс мог решать несколько задач, например выполнять функции контроллера и роутера. Сейчас за роутинг отвечает отдельный класс, контроллеры в отдельном классе, ответы на запросы в 3м. Постарался сделать как можно проще, что бы соответсвовало парадигме KISS

irbis45 committed:
hw9
# локализация
# получение локали по умолчанию из браузера
# попытка реализовать middleware на PSR-7, PSR-15. На Laravel 12 нормально не работает
